crates/fiber-lib/src/build.rs

```diff
@@ -107,7 +109,11 @@ fn main() {
             }
         }
     }
-
+    if latest_db_version.is_empty() {
+        // If there is no migrations, `latest_db_version` is set to today.
```

Comment:
We can't do that here, let's say the user is running on date A, but we add a migration script before date A (which hasn't been released yet) to our development branch, then new version is released, the migration will be ignored.